tracing-log/src/lib.rs

```diff
@@ -143,10 +138,6 @@ use tracing_core::{
 #[cfg_attr(docsrs, doc(cfg(feature = "log-tracer")))]
 pub mod log_tracer;
 
-#[cfg(feature = "trace-logger")]
-#[cfg_attr(docsrs, doc(cfg(feature = "trace-logger")))]
```

Comment:
since this module had a feature flag, can we also remove the feature flag from the `Cargo.toml`?

Reply:
whoops, sorry—I had completely missed that.